lib/export/coldp.rb

```diff
@@ -27,6 +27,26 @@ def self.otus(otu_id)
         .where('(otus.name IS NULL) OR (otus.name = taxon_names.cached)')
     end
 
+    def self.project_members(project_id)
+      project_members = {}
+      ProjectMember.where(project_id: project_id).each do |pm|
+        if pm.user.orcid.nil?
+          project_members[pm.user_id] = pm.user.name
+        else
+          project_members[pm.user_id] = pm.user.orcid.gsub('orcid.org/', '')  # orcid.org domain likely not expected by the checklistbank importer?
```

Comment:
Is there a standard?  I expect our `cached` to match a standard.  Jim was very good at working this out, but we might have added ORCiD after he left.

Reply:
It looks like on sfg, the orcids are actually stored as either:

https://orcid.org/0000-0000-0000-0000  (205 records)
orcid.org/0000-0000-0000-0000  (11 records)

At least in the metadata model of ChecklistBank, orcid IDs are provided like 0000-0000-0000-0000: http://api.checklistbank.org/dataset/2144.json

If ChecklistBank ever adds other people resolvers, they're going to need to namespace the IDs so maybe it's better to leave it in.